src/main/java/TaskList.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Arrays;
+
+public class TaskList {
+    private Task[] tasks;
+    private static int tasksCount = 0;
+
+    public TaskList (int expectedTasksCount) {
+        this.tasks = new Task[expectedTasksCount];
+    }
+
+
+    public Task[] getTasks() {
+        return Arrays.copyOfRange(this.tasks, 0,  tasksCount);
+    }
+
+    public int getTasksCount() {
+        return tasksCount;
+    }
+
+    public String toString() {
+        int index = 1;
+        StringBuilder output = new StringBuilder();
+        for (Task task: tasks) {
```

Comment:
Good job for not using single character integers in your for-loop! a negative example would be 
for(int a=0;a<tasks.length;a++){
...
}